ETC.tsv

```diff
@@ -4340,7 +4340,7 @@ ETC_20150317_004339	Requires iCoin.{nl}Are you sure you want to reset?
 ETC_20150317_004340	{@st59} Increase in Physical Attack and Critical Attack{/}
 ETC_20150317_004341	{@st59} Increase in HP, HP Recovery rate, Critical Resistance and Max Inventory Slots{/}
 ETC_20150317_004342	{@st59} Increase in Magic Damage and Block Penetration{/}
-ETC_20150317_004343	{@st59} Increase in Accuracy, Evasion and Critical Attack {/}
+ETC_20150317_004343	{@st59} Increase in Accuracy, Evasion and Critical Rate {/}
```

Comment:
I forgot to tell you. This was because of the previous pull request that wasn't supposed to be there. I was afraid imc would override your change.